### ISSUE TITLE
Unable to authenticate after updating home assisant. 

### ISSUE THREAD

**Issue by chiments** [JeffSteinbok/hass-dreo#69]
Upgraded to 2024.1.5 and the integration broke. I reverted back to 2023.11.3 and the problem followed.  I am unable to sign in from Home Assistant. I can sign in fine on the Dreo app.  I've got 3 entries in the logs. Did the API server change?

This error originated from a custom integration.

Logger: dreo
Source: custom_components/dreo/__init__.py:38
Integration: Dreo (documentation, issues)
First occurred: 5:31:50 PM (1 occurrences)
Last logged: 5:31:50 PM

Unable to login to the dreo server

This error originated from a custom integration.

Logger: pydreo
Source: custom_components/dreo/pydreo/helpers.py:166
Integration: Dreo (documentation, issues)
First occurred: 5:31:48 PM (1 occurrences)
Last logged: 5:31:48 PM

No response from API

This error originated from a custom integration.

Logger: pydreo
Source: custom_components/dreo/pydreo/__init__.py:237
Integration: Dreo (documentation, issues)
First occurred: 5:31:48 PM (1 occurrences)
Last logged: 5:31:48 PM

Error logging in with username and password

**Comment by JeffSteinbok** [comment]
Have you upgraded to latest dreo integration version?

-Jeff

On Jan 20, 2024, at 3:38 PM, chiments ***@***.***> wrote:

﻿

Upgraded to 2024.1.5 and the integration broke. I reverted back to 2023.11.3 and the problem followed. I am unable to sign in from Home Assistant. I can sign in fine on the Dreo app. I've got 3 entries in the logs. Did the API server change?

This error originated from a custom integration.

Logger: dreo
Source: custom_components/dreo/init.py:38
Integration: Dreo (documentation, issues)
First occurred: 5:31:50 PM (1 occurrences)
Last logged: 5:31:50 PM

Unable to login to the dreo server

This error originated from a custom integration.

Logger: pydreo
Source: custom_components/dreo/pydreo/helpers.py:166
Integration: Dreo (documentation, issues)
First occurred: 5:31:48 PM (1 occurrences)
Last logged: 5:31:48 PM

No response from API

This error originated from a custom integration.

Logger: pydreo
Source: custom_components/dreo/pydreo/init.py:237
Integration: Dreo (documentation, issues)
First occurred: 5:31:48 PM (1 occurrences)
Last logged: 5:31:48 PM

Error logging in with username and password
